app/pages/angelfish.tsx

```diff
@@ -48,7 +48,7 @@ const Home: React.FC<HomePageProps> = (props: HomePageProps) => {
         </section>
         <section>
           <div className="page-container">
-            <div className="doses-data-grid">
+            <div className="doses-data-grid" key={region}>
```

Comment:
Forces remount so the bars re-animate - @jishaal any worries with performance of this?

Reply:
Nah should be fine imo 👌🏼